lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -811,6 +811,59 @@ ExtractSwiftTypeFromCxxInteropType(CompilerType type, TypeSystemSwift &ts,
   return {};
 }
 
+
+/// Synthetic child that wraps a value object.
```

Comment:
... that wraps a TypeSystemClang value object?

Reply:
Actually this is a generic implementation, which is used for both wrapping Clang types in Swift and Swift types in C++